app/controllers/bill_controller.rb

```diff
@@ -0,0 +1,26 @@
+class BillController < ApplicationController
+
+  def index
+    @bills = Bill.all
+  end
+
+  def new
+    @bill = Bill.new
+  end
+
+  def create
+  end
+
+  def show
+    @bill = Bill.find(params[:id])
+  end
+
+  def edit
+    @bill = Bill.find(params[:id])
+  end
+
+  def update
```

Comment:
Update the bill would be used to add items (second round of drinks) or delete (food came out wrong, comp the item)